docs/source/building.rst

```diff
@@ -547,6 +547,12 @@ The following variables are makefile variables, not environment variables.
     Only supported by the Aldec Riviera-PRO, Aldec Active-HDL, and Siemens EDA Questa simulators.
 
 
+.. make:var:: PYTHON_BIN
+
+    The path to the ``python`` binary.
+    Set to the result of ``cocotb-config --python-bin`` if present on the ``PATH``.
+    Otherwise defaults to ``python``.
```

Comment:
It's beyond the scope of this PR, but should we not default to `python3`? (I think it was/is the recommendation to not expect `/usr/bin/python` to be Python 3, and last I checked macos still shipped Python 2 as `/usr/bin/python`